src/Helpers/PageHelpers/IPageHelper.cs

```diff
@@ -13,7 +13,7 @@ public interface IPageHelper
 
         void SaveAnswers(Dictionary<string, dynamic> viewModel, string guid, string form, IEnumerable<CustomFormFile> files, bool isPageValid, bool appendMultipleFileUploadParts = false);
 
-        void SaveCaseReference(string guid, string caseReference);
+        void SaveCaseReference(string guid, string caseReference, bool isGenerated=false, string generatedRefereceMappingId="");
```

Comment:
We need spaces around the equals on these default values

Reply:
Done